sechub-doc/src/docs/asciidoc/documents/shared/configuration/sechub_config_example21_webscan_login_form_with_totp.json

```diff
@@ -0,0 +1,16 @@
+{
+  "apiVersion" : "1.0",
+  "webScan" : {
+    "url" : "https://productfailure.demo.example.org",
+    "login" : {
+      "url" : "https://productfailure.demo.example.org/login",
+      "form" : { ... },
```

Comment:
Please either
- remove the form element or 
- create empty element with "//..." part inside (should be accepted by output formatter)
- add a simple example there.

Reason: I did not check it, but the output formatter in asciidoc output should have probllems here, because it is not valid json

Reply:
Additionally: we have currently no unit test which reads a sechub configuration with totp setup inside. It would be nice, if this document file would be read by a test and checked that it contains the expected totp information and that it is valid json.

if it is not... the documentation would contain a invalid example file

So please add a unit test which reads the file and does some simple checks.